cc26xx_cc13xx: can't launch debug server.

<!--
====================================
IF YOUR ISSUE IS RELATED TO SECURITY
====================================
please submit it to the security mailing-list security@riot-os.org.

If your issue is a question related to the usage of RIOT, please submit it to
the user mailing-list users@riot-os.org or to the developer mailing-list
devel@riot-os.org.
-->

#### Description
<!--
Example: Cannot build gnrc_networking application for samr21-xpro board.
-->

Cannot launch the debug server (`gdb_agent_console`) on boards using the `makefiles/tools/uniflash.inc.mk` script. It seems that this script has been removed from Uniflash version 5.2.0 and resides now on the [Code Composer Studio IDE][link-to-editor].

[link-to-editor]: https://www.ti.com/tool/CCSTUDIO

#### Steps to reproduce the issue
<!--
Try to describe as precisely as possible here the steps required to reproduce
the issue. Here you can also describe your hardware configuration, the network
setup, etc.
-->

Use `make debug-server` on any example and specifying a board that uses the CC26xx/CC13xx processors, for example the `cc1352-launchpad`.

#### Expected results
<!--
Example: The gnrc_networking application builds on samr21-xpro.
-->
Getting the debugging server running.

#### Actual results
<!--
Please paste or specifically describe the actual output.
-->
```
Debug server program /home/jeandudey/.local/opt/ti/uniflash_5.2.0/ccs_base/common/uscif/gdb_agent_console is required but not found in PATH.  Aborting.
/home/jeandudey/Dev/RIOT/examples/hello-world/../../Makefile.include:704: recipe for target 'debug-server' failed
make: *** [debug-server] Error 1
```
#### Versions
<!--
Operating system: Mac OSX, Linux, Vagrant VM
Build environment: GCC, CLang versions (you can run the following command from
the RIOT base directory: make print-versions).
-->
```
Operating System Environment
-----------------------------
         Operating System: "Ubuntu" "18.04.3 LTS (Bionic Beaver)"
                   Kernel: Linux 5.0.0-37-generic x86_64 x86_64
             System shell: /bin/dash (probably dash)
             make's shell: /bin/dash (probably dash)

Installed compiler toolchains
-----------------------------
               native gcc: gcc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
        arm-none-eabi-gcc: arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 9-2019-q4-major) 9.2.1 20191025 (release) [ARM/arm-9-branch revision 277599]
                  avr-gcc: missing
         mips-mti-elf-gcc: missing
               msp430-gcc: missing
     riscv-none-embed-gcc: missing
     xtensa-esp32-elf-gcc: missing
   xtensa-esp8266-elf-gcc: missing
                    clang: clang version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)

Installed compiler libs
-----------------------
     arm-none-eabi-newlib: "3.1.0"
      mips-mti-elf-newlib: missing
  riscv-none-embed-newlib: missing
  xtensa-esp32-elf-newlib: missing
xtensa-esp8266-elf-newlib: missing
                 avr-libc: missing (missing)

Installed development tools
---------------------------
                   ccache: ccache version 3.4.1
                    cmake: cmake version 3.10.2
                 cppcheck: missing
                  doxygen: missing
                      git: git version 2.17.1
                     make: GNU Make 4.1
                  openocd: Open On-Chip Debugger 0.10.0+dev-01012-ged8fa09c (2020-01-15-14:01)
                   python: Python 2.7.17
                  python2: Python 2.7.17
                  python3: Python 3.6.9
                   flake8: error: /usr/bin/python3: No module named flake8
               coccinelle: missing
```

<!-- Thanks for contributing! -->

--- a/makefiles/tools/uniflash.inc.mk
+++ b/makefiles/tools/uniflash.inc.mk
@@ -5,7 +5,7 @@ UNIFLASH_CONFIG_CCXML ?= $(BOARDSDIR)/$(BOARD)/dist/$(CPU_MODEL)_$(XDEBUGGER).cc
 UNIFLASH_CONFIG_DAT ?= $(BOARDSDIR)/$(BOARD)/dist/$(CPU_MODEL)_$(XDEBUGGER).dat
 UNIFLASH_CONFIG_GDB ?= $(BOARDSDIR)/$(BOARD)/dist/$(CPU_MODEL)_gdb.conf
 
-export UNIFLASH_PATH ?= "UNIFLASH_PATH unconfigured"
+UNIFLASH_PATH ?= "UNIFLASH_PATH unconfigured"
 # check which uniflash version is available, either 4.x or 3.x
 ifneq ("$(wildcard $(UNIFLASH_PATH)/dslite.sh)","")
   FLASHER ?= $(UNIFLASH_PATH)/dslite.sh

--- a/makefiles/tools/uniflash.inc.mk
+++ b/makefiles/tools/uniflash.inc.mk
@@ -24,8 +24,10 @@ else
   RESET ?= $(UNIFLASH_PATH)/uniflash.sh
   RESET_FLAGS ?= -ccxml $(UNIFLASH_CONFIG_CCXML) -reset
 endif
+
+CCS_PATH ?= "CCS_PATH unconfigured"
 # configure the debug server
-DEBUGSERVER = $(UNIFLASH_PATH)/ccs_base/common/uscif/gdb_agent_console
+DEBUGSERVER = $(CCS_PATH)/ccs/ccs_base/common/uscif/gdb_agent_console
 DEBUGSERVER_FLAGS = -p 3333 $(UNIFLASH_CONFIG_DAT)
 
 # configure the debugging tool

